res/satis-schema.json

```diff
@@ -0,0 +1,115 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "name": "Repository Configuration",
+    "type": "object",
+    "additionalProperties": false,
+    "required": [ "name", "homepage" ],
+    "properties": {
+        "name": {
+            "type": "string",
+            "description": "Repository name."
+        },
+        "description": {
+            "type": "string",
+            "description": "Short Repository description."
+        },
+        "homepage": {
+            "type": "string",
+            "description": "Homepage URL for the project.",
+            "format": "uri"
+        },
+        "require": {
+            "type": "object",
+            "description": "This is a hash of package name (keys) and version constraints (values) that are required to run this package.",
+            "additionalProperties": true
```

Comment:
rather than ``true``, this should use a schema object here, to force all additional properties to contain strings